mail/postfix/src/opnsense/mvc/app/models/OPNsense/Postfix/Address.xml

```diff
@@ -0,0 +1,21 @@
+<model>
+    <mount>//OPNsense/postfix/address</mount>
+    <description>Postfix address rewrite configuration</description>
+    <version>1.0.0</version>
+    <items>
+        <addresses>
+                <address type="ArrayField">
+                        <enabled type="BooleanField">
+                                <default>1</default>
+                                <Required>Y</Required>
+                        </enabled>
+                        <from type="TextField">
+                                <Required>Y</Required>
+                        </from>
+                        <to type="TextField">
+                                <Required>Y</Required>
+                        </to>
+            </address>
```

Comment:
please check the indent, you probably indented everything here in address twice (8 spaces) and also the `address` does not match.

Reply:
Fixed